chapters/image_accceses.md

```diff
@@ -0,0 +1,97 @@
+# Image Accesses
+
+SPIR-V has various [Image Instructions](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_image_instructions) that will interact with `OpTypeImage`. This chapter aims to give some extra context around these instructions.
```

Comment:
I would recommend removing the word "will" ->

"SPIR-V has various Image Instructions that ~~will~~ interact with `OpTypeImage`."

I think it is accurate to say the images work now, so the present tense is confusing.